versioned_docs/version-8.1/components/best-practices/best-practices-overview.md

```diff
@@ -2,11 +2,11 @@
 title: Overview
 ---
 
-The Camunda best practices are our condensed experience of using BPMN and DMN on the Camunda toolstack, and collected by consulting engagement with our customers, feedback from the community and various other occasions. Best Practices are a mix of conceptual and practical implementation information.
+The Camunda best practices are our condensed experience of using BPMN and DMN on the Camunda toolstack, and collected by consulting engagement with our customers, feedback from the community, and various other occasions. Best Practices are a mix of conceptual and practical implementation information.
```

Comment:
The only comment I have here is consistency of uppercase "Best Practices" 👍

Reply:
@christinaausley : if I get your comment correctly, you'd change these all to upper case? I'm not quite sure how to fix all, because some should IMHO stay lower case, e.g. "Note that Camunda give the same guarantee as the core product for best practices"

I'd personally prefer to switch all to lower case "p" but you surely have a better overview and judgement.

So I propose to merge this PR and then you can fix the upper/lower case. Alternatively, if we switch to lower case I can fix the two places. 

How do we proceed?

Reply:
@tobiasschaefer -- Good questions. Given Camunda Best Practices are a company component of their own (like Guides, for example,) we do indeed uppercase Best Practices across our documentation when referring to this content. If we could uppercase in this instance, we'll be set to merge. Let me know if you'd like me to commit this for you or if you have any questions👍

Reply:
@christinaausley : Thanks for your feedback. I unified it to "Best Practices". From my point of view this PR is ready to merge.